import/repos.json

```diff
@@ -124,8 +124,8 @@
     "repo": "https://github.com/EventStore/EventStore-Client-Go.git",
     "branches": [
       {
-        "version": "1.0.0",
-        "name": "master"
+        "version": "1.0.2",
+        "name": "persistent_subscriptions_snippets_naming"
```

Comment:
**TODO:** Update it once https://github.com/EventStore/EventStore-Client-Go/pull/97 is merged.